src/components/Modal/Modal.tsx

```diff
@@ -0,0 +1,76 @@
+import React, {useEffect, useState} from "react";
+import {ModalProps} from "./types";
+import Button from "../Button/Button";
+import ModalStyles from "./Modal.styles"
+import {iconTypes} from "../Icon/collection";
+
+const { ModalHeader, ModalStyled, ModalFooter, ModalContent} = ModalStyles;
+
+const Modal: React.FC<ModalProps> = ({
+    id = String(Date.now()),
+    children,
+    isVisible,
+    cancelDisabled,
```

Comment:
isCancelDisbaled might be more consistent... sorry

Reply:
cancelDisabled -> isCancelDisabled
okDisabled -> isOkDisabled

fixed, pushed and merged